### DISCUSSION TITLE
Cannot open xournalpp files pulled from git - "Error opening file: Document must begin with an element (e.g. <book>)"

### DISCUSSION THREAD

**Discussion by adam-peter** [xournalpp/xournalpp#5886] [General]
My problem is that I started a git repo where I versioned all my .xopp files (with git-lfs) on my machine at home (which is running arch and gnome). I tried opening the file with xournal on my laptop (running arch & hyprland) and I get a confusing error.

It doesn't help me much, only thing I know is that I can't open the file.

The weird thing is that I tried to open it on my mac m1 aswell, getting the same error. But when I cloned the repo in a different place on the SAME original computer, on which the notes were written, I got no errors and the file opened as expected.

Do you have any idea on why this might be happening? I'm sending a screenshot of the error and a [link to a sample repo](https://gitlab.com/adam-peter/xournal-error-repo) I made public, with files with the same issue.

![240924_16h00m25s_screenshot](https://github.com/user-attachments/assets/2009f697-d475-40e2-8589-85b58ad4572b)

PS: I'm not sure if it might be a factor, but I prefix all my file names with numbers. Just realized that when creating the test file for the error sample repo - so I prefixed it with a number aswell, just in case.

**Comment by GithubAnon0000** [comment]
On debian oldstable with xournalpp 1.2.3 I can open your file just fine.

Maybe you have several different xournalpp versions on your devices and your file needs a recent version?
